src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,68 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module _Drawer {
+  export class ErrorBar extends Element {
+    public static ERROR_BAR_CLASS = "error-bar";
+    public static ERROR_BAR_MIDDLE_CLASS = "error-bar-middle";
+    public static ERROR_BAR_LOWER_CLASS = "error-bar-lower";
+    public static ERROR_BAR_UPPER_CLASS = "error-bar-upper";
+
+    private _errorTickRadius = 10;
```

Comment:
I didn't want to say tickLength because this is actually half of the tickLength, which could be confusing. I could switch this to tickRadius and add a /2 in the calculations below, do you think that would be better? I agree radius is kind of the wrong word to use here.